tests/qam-updinstall/update_install.pm

```diff
@@ -92,15 +97,17 @@ sub run {
     my ($self) = @_;
     my $incident_id = get_required_var('INCIDENT_ID');
     my $repos = get_required_var('INCIDENT_REPO');
+    my %installable;    #Binaries already released that can already be installed.
+    my @new_binaries;    #Binaries introduced by the update that will be installed after the repos are added.
```

Comment:
Tidy was failing because of this, which is right now wrong comment, I think ...
I spit on tidy and hope I will not have to use it ever again :pray: